src/Branching/perform-migration-from-svn-to-git.md

```diff
@@ -0,0 +1,183 @@
+
+### Clone the Subversion repository using git-svn
+
+The following command will do the standard git-svn transformation using the authors-transform.txt file created in the previous step. It will place the Git repository in the "c:\mytempdir" folder in your local machine.
+```
+git svn clone ["SVN repo URL"] --prefix=svn/ --no-metadata --authors-file "authors-transform.txt" --stdlayout c:\mytempdir
+```
+
+> [!NOTE]
+>
+> The –prefix=svn/ is necessary because otherwise the tools can’t tell apart SVN revisions from imported ones. Setting a prefix (with a trailing slash) is strongly encouraged in any case, as your SVN-tracking refs will then be located at "refs/remotes/$prefix/", which is compatible with Git’s own remote-tracking ref layout (refs/remotes/$remote/).
+>
+> Setting a prefix is also useful if you wish to track multiple projects that share a common repository. By default, the prefix is set to origin/.
+
+If you are using the standard trunk, branches, tags layout you’ll just put –stdlayout. However if you have something different you may have to pass the –trunk, –branches, and –tags in to identify what is what. For example if your repository structure was trunk/companydir and you branched that instead of trunk, you would probably want ‘–trunk=trunk/companydir –branches=branches‘.
+
+```
+git svn clone ["SVN repo URL"] --prefix=svn/ --no-metadata --trunk=/trunk --branches=/branches --tags=/tags  --authors-file "authors-transform.txt" c:\mytempdir
+```
+
+> [!NOTE]
+>
+> This command can take a few minutes to several hours depending on the size of the SVN repository. Upon completion, you will have a Git checkout of your repository.
+
+### Convert version control-specific configurations
+
+If your svn repo was using svn:ignore properties, you can  convert this to a .gitignore file using:
+```
+cd c:\mytempdir
+git svn show-ignore > .gitignore
+git add .gitignore
+git commit -m 'Convert svn:ignore properties to .gitignore.'
+```
+> [!TIP]
+>
+> Read more about .gitignore [Ignore file changes with Git](
+https://docs.microsoft.com/en-us/vsts/git/tutorial/ignore-files?tabs=visual-studio)
+
+### Push repository to a bare git repository
+
+In this step, you will create a bare repository and make its default branch match SVN's trunk branch name.
+
+1. Create a bare Git repository
+
+```
+git init --bare c:\new-bare.git
+cd c:\new-bare.git
+git symbolic-ref HEAD refs/heads/trunk
+```
+2. Push the local Git repository to the new bare Git repository
+```
+cd c:\mytempdir
+git remote add bare c:\new-bare.git
+git config remote.bare.push 'refs/remotes/*:refs/heads/*'
+git push bare
+```
+3. Rename "trunk" branch to "master"
+Your main development branch will be named “trunk” which matches the name it was in Subversion. You’ll want to rename it to Git’s standard “master” branch using:
```

Comment:
Unicode quotes here - should be standard ascii.  (And perhaps should be code blocked).

Reply:
done